File: src/main/java/no/nav/domain/pensjon/regler/gomregning/BeregningerFaktoromregningResultat.java
package no.nav.domain.pensjon.regler.gomregning;

import no.nav.domain.pensjon.regler.BatchStatus;
import no.nav.domain.pensjon.regler.beregning.Beregning;

import java.io.Serializable;
import java.util.ArrayList;
import java.util.List;

public class BeregningerFaktoromregningResultat implements Serializable {

    
    private Long vedtakId;
    private BatchStatus batchStatus;
    private List<Beregning> beregningResultatListe;

}

File: src/main/kotlin/no/nav/pensjon/regler/domain/gomregning/BeregningerFaktoromregningResultat.kt
package no.nav.pensjon.regler.domain.gomregning

import no.nav.pensjon.regler.domain.BatchStatus
import no.nav.pensjon.regler.domain.beregning.Beregning
import java.io.Serializable

class BeregningerFaktoromregningResultat : Serializable {
    var vedtakId: Long? = null
    var batchStatus: BatchStatus? = null
    var beregningResultatListe: MutableList<Beregning> = mutableListOf()
}